src/main/java/com/cinema/model/Order.java

```diff
@@ -0,0 +1,62 @@
+package com.cinema.model;
+
+import java.time.LocalDateTime;
+import java.util.List;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.ManyToOne;
+import javax.persistence.OneToMany;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "orders")
+public class Order {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @OneToMany
+    private List<Ticket> tickets;
+    private LocalDateTime orderDate;
+    @ManyToOne
```

Comment:
<img width="633" alt="Screenshot 2020-10-12 at 13 13 49" src="https://user-images.githubusercontent.com/68664175/95734902-c67d7e80-0c8c-11eb-8077-83612430962a.png">

Reply:
FetchType.Eager can be used with ManyToOne connection

Reply:
Agree, sorry for confusing